src/main/resources/org/padaiyal/wordspackage/WordCounter.properties

```diff
@@ -0,0 +1 @@
+string.regex.wordSeparator=\\s+
```

Comment:
shouldn't naming format of the properties file be in lower case?

Reply:
Not necessarily.